lib/app_colors.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  static const Color chineseBlack = Color(0xff15151A);
```

Comment:
Optional: I think you can generate color from xml assets as well by https://pub.dev/packages/flutter_gen 🔥